api/events/schema.py

```diff
@@ -305,3 +315,23 @@ def resolve_all_events(self, info, **kwargs):
                                      pages=pages)
 
         return PaginatedDailyRoomEvents(DailyRoomEvents=all_days_events)
```

Comment:
This return statement should be tested as well

Reply:
All the Return statements are tested @Arusey 
<img width="703" alt="Screenshot 2019-06-17 at 14 49 24" src="https://user-images.githubusercontent.com/39084014/59602368-36b6d700-910f-11e9-848f-b19da1bab5fc.png">